cmd/admin-handlers-config-kv.go

```diff
@@ -309,7 +309,12 @@ func (a adminAPIHandlers) RestoreConfigHistoryKVHandler(w http.ResponseWriter, r
 		return
 	}
 
-	delServerConfigHistory(ctx, objectAPI, restoreID)
+	if err = delServerConfigHistory(ctx, objectAPI, restoreID); err != nil {
```

Comment:
A returning handler is Always a success you are not fixing anything here @jiuker

Reply:
We on purpose do not want to return an error for config history..